lib/ansible/modules/user.py

```diff
@@ -176,6 +176,12 @@
             - If no passphrase is provided, the SSH key will default to having no passphrase.
         type: str
         version_added: "0.9"
+    ssh_key_trials:
```

Comment:
Would `ssh_key_rounds` be a better name (`ssh-keygen -A [-a rounds]`)?

Reply:
ssh_key_rounds would be OK for me.